### DISCUSSION TITLE
Master warn / Master caution

### DISCUSSION THREAD

**Discussion by pawingnut44** [flybywiresim/aircraft#3750] [Help]
Hello,

Both the master warn and master caution reset buttons are now unresponsive for me, and i can't seem to find a reason.  any fixes for this? thanks.

**Comment by hotshotp** [comment]
probably a mod conflict, please try reinstalling

**Comment by Paytheon** [comment]
I confirm the master caution reset is not function. I have no mods.

**Comment by hotshotp** [comment]
can you show a video of it not working?

**Comment by pawingnut44** [comment]
I will once I get on after bit .

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by pawingnut44** [comment]
btw, it does not work in the DEV verison, I switched to the stable earlier
today and they functioned fine.

<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=icon>
Virus-free.
www.avast.com
<https://www.avast.com/sig-email?utm_medium=email&utm_source=link&utm_campaign=sig-email&utm_content=webmail&utm_term=link>
<#DAB4FAD8-2DD7-40BB-A1B8-4E2AA1F9FDF2>

**Comment by pawingnut44** [comment]
my video attemping to silence master warn. also same effect for master caution, no success.  Did switch to stable version  from dev verion, it does function there.

https://youtu.be/mYaXIQg-2hQ

**Reply by hotshotp** [reply]
were you overspeeding? you are not able to silence the overspeed alarm in the real plane
